main.nf

```diff
@@ -775,7 +776,8 @@ process ConcatVCF {
 	done
 	cat header raw_calls > unsorted.vcf
 	java -jar \${PICARD_HOME}/picard.jar SortVcf I=unsorted.vcf O=$outputFile
-    rm unsorted.vcf
+	rm unsorted.vcf
+	gzip -v $outputFile
```

Comment:
Looks not this one:
`$ java -jar ${PICARD_HOME}/picard.jar SortVcf I=bugger.vcf OUTPUT=/dev/stdout 
[Wed Mar 22 14:42:27 CET 2017] picard.vcf.SortVcf INPUT=[bugger.vcf] OUTPUT=/dev/stdout    VERBOSITY=INFO QUIET=false VALIDATION_STRINGENCY=STRICT 
`
...


`Exception in thread "main" java.lang.IllegalArgumentException: Must specify file or stream output type.
        at htsjdk.variant.variantcontext.writer.VariantContextWriterBuilder.build(VariantContextWriterBuilder.java:407)
`